src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  @for $i from 1 through 5 {
+    $active: if($i <= 4, true, false);
+    $star-image: if($active, "star-active", "star");
+
+    .stars__star:nth-child(#{$i}) {
+      background-image: url(../images/#{$star-image}.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+      @include square(16px);
+      margin-right: 4px;
+    }
+  }
```

Comment:
Very strange logic and actually it doesn't work.
By default all stars are inactive. The count of active stars should depend on the modifier, for example, `stars--5` should mark five stars as active.

```scss
@for $i from 0 through 5 {
    &--#{$i} {
      & :nth-child(-n + #{$i}) {
        background-image: url(../images/star-active.svg);
      }
    }
  }
```

- Only background-image should change, the same as you had before
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/bb70d783-71c3-4ff1-b67d-e93f0ed90ac4)

- use scss nesting, you don't need to duplicate selectors. For example `&__star`